bcgov/cas-registration
# chore: add FuelType model and populate table

Card: https://github.com/bcgov/cas-reporting/issues/180

Adding FuelType model and populating table in the erc schema. Also updated the erd diagram to accommodate changes.

To test:

To test:

    Navigate to /bc_obps/ and run the following commands:
    make install_dev_tools
    make install_poetry_deps
    make start_pg
    make create_db
    make migrate
    make run

    In another terminal navigate to /bc_obps/ and run the shell with the command python manage.py shell

    Run these commands in the shell
    from reporting.models import FuelType
    FuelType.objects.count()

This should return the number of items, 88
